docs/src/whatsnew/3.1.rst

```diff
@@ -180,6 +179,9 @@ This document explains the changes made to Iris for this release
 #. `@MHBalsmeier`_ updated and simplified non-conda installation on Debian-based distros.
    (:pull:`4260`)
 
+#. `@bjlittle`_ updated the ``intersphinx_mapping`` and fixed documentation
+   to use ``stable`` URLs for `matplotlib`_. (:pull:`4003`)
+
```

Comment:
I'm a bit confused by this...it makes sense that its in the 3.1. whats new, but we also included it in the 3.0.4 release so shouldn't it also be included in the 3.0 whats new?

Reply:
No, I **_think_** that is what the `[pre-3.1.0]` labels were for.
If I get it right, they mark changes which was brought in from main + included in (e.g.) 3.0.4, because it was ~too hard to merge in the fixes without, but strictly aren't part of a bugfix and so belong with the 3.1 updates.

Reply:
Though, I suppose we could have it in **_both_** places ???

Reply:
I'm still a bit conflicted about this (in future maybe we write down the intention behind these sort of labels, to avoid confusion!) but as they only concern internal changes/a small docs change, I'm not majorly concerned by it and happy to just keep them in the 3.1 whats new